### DISCUSSION TITLE
What is the state of babel plugin compatibility?

### DISCUSSION THREAD

**Discussion by agriffis** [swc-project/swc#7103] [Q&A] [answered]
This PR looks really neat:

https://github.com/swc-project/swc/pull/1465

It's not clear to me what happens next. Is it possible now to use babel plugins with swc, and if so, how does one do that?

**Comment by kwonoj** [comment, marked as answer]
> Is it possible now to use babel plugins with swc, and if so, how does one do that?

Having an ast translator is not directly related with using babel plugins in swc. It is an internal to help interop some cases currently.
